ferretdb_packaging/development.Dockerfile

```diff
@@ -0,0 +1,60 @@
+# syntax=docker/dockerfile:1
+
+ARG POSTGRES_VERSION=16
+
+FROM postgres:${POSTGRES_VERSION} AS development
+
+ARG DOCUMENTDB_VERSION
+
+# redeclare arg to use within build
+# https://docs.docker.com/reference/dockerfile/#understand-how-arg-and-from-interact
```

Comment:
longer explanation https://qmacro.org/blog/posts/2024/05/13/using-arg-in-a-dockerfile-beware-the-gotcha/